client/trino-jdbc/src/main/java/io/trino/jdbc/KnownTokenCache.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.jdbc;
+
+import io.trino.client.auth.external.KnownToken;
+
+public enum KnownTokenCache
+{
+    NONE {
+        @Override
+        KnownToken create()
+        {
+            return KnownToken.local();
```

Comment:
inline this `local()` method.

Reply:
I don't want to, as this would require declaring KnownToken classes instances with public modifiers etc. I want them to be hidden in the client-api and here used only through designated factory methods, without revealing their implementation etc.